build/ci/build.yml

```diff
@@ -10,7 +10,7 @@ parameters:
   windowsImage: 'windows-latest'                            # the name of the Windows VM image
 
   # Tool Parameters
-  dotnetVersion: '8.0.101'                                  # the version of .NET to use
+  dotnetVersion: '8.0.200'                                  # the version of .NET to use
```

Comment:
maybe use 8.0.201 SDK to avoid [workload install issues](https://github.com/dotnet/maui/issues/20600)?